src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/commands.py

```diff
@@ -174,6 +174,10 @@ def load_command_table(self, _):  # pylint: disable=too-many-statements
         g.command('list', 'acr_task_list')
         g.command('delete', 'acr_task_delete')
         g.command('update', 'acr_task_update')
+        g.command('credential add', 'acr_task_credential_add')
+        g.command('credential update', 'acr_task_credential_add')
+        g.command('credential delete', 'acr_task_credential_delete')
```

Comment:
I think `remove` is the right word here, but will defer to @northtyphoon

Reply:
According to the English stackexchange link, `delete` makes sense here because we don't store that credential once they delete it. 
There are other commands in the `acr` that use `delete` except for `acr network-rule remove`